plugins/azure/appservice/privateEndpointsEnabled.js

```diff
@@ -0,0 +1,51 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Web Apps Private Endpoints Configured',
+    category: 'App Service',
+    domain: 'Application Integration',
+    description: 'Ensures thatWeb Apps are accessible only through private endpoints',
+    more_info: 'Azure Private Endpoint is a network interface that connects you privately and securely to a service powered by Azure Private Link. Private Endpoint uses a private IP address from your VNet and provides secure connectivity between clients on private network and app',
```

Comment:
```suggestion
    more_info: 'Enabling private endpoints for Azure App Service enhances security by allowing access exclusively through a private network, minimizing the risk of public internet exposure and protecting against external attacks.',
```
should be rephrased to something like this